src/modules/map/components/CompareSlider.vue

```diff
@@ -39,9 +41,40 @@ export default {
     },
     mounted() {
         this.compareRatio = this.storeCompareRatio
+        this.slice()
     },
     methods: {
         ...mapActions(['setCompareRatio']),
+        slice() {
+            if (olLayer) {
+                olLayer.un('prerender', this.onPreCompose)
+                olLayer.un('postrender', this.onPostCompose)
+            }
+            olLayer = this.getMap()
+                .getAllLayers()
+                .find((layer) => {
+                    return (
+                        this.visibleLayerOnTop && layer.get('id') === this.visibleLayerOnTop.getID()
+                    )
+                })
+            if (olLayer) {
+                olLayer.on('prerender', this.onPreCompose)
+
+                olLayer.on('postrender', this.onPostCompose)
+            }
+        },
+        onPreCompose(event) {
+            const ctx = event.context
+            const width = ctx.canvas.width * this.compareRatio + 20
```

Comment:
it might be good to explain why this 20 (I imagine it's the half width of the element with the handle?)

Reply:
I'll put a comment to inform about this and yes, this is to allow the layer to reach the red bar of the component.